hop/models.py

```diff
@@ -148,41 +148,41 @@ def from_email(cls, email_input):
         )
 
     @classmethod
-    def from_email_file(cls, filename):
+    def load_file(cls, filename):
         """Create a new GCNCircular from an RFC 822 formatted circular file.
 
         Args:
-            filename: the GCN filename
+            filename: The GCN filename.
 
         Returns:
             The GCNCircular.
 
         """
-
         with open(filename, "r") as f:
-            return cls.from_email(f)
+            return cls.load(f)
 
 
 @dataclass
 class MessageBlob(object):
     """Defines an unformatted message structure.
 
-    This is included as a dataclass to mirror the implementation of structured formats.
+    This is included to mirror the implementation of structured formats.
 
     """
 
     content: str
+    missing_schema: bool = False
 
     def asdict(self):
         """Represents the message as a dictionary.
 
         Returns:
-            dict: the dict representation of the message
+            The dictionary representation of the message.
 
         """
         return asdict(self)
 
-    def wrap_message(self):
+    def serialize(self):
```

Comment:
I believe that `wrapped_message` should have `content` be assigned to `self.content`:
```
wrapped_message = {"format": "blob", "content": self.content}
```

Otherwise, `subscribe.print_message` fails when handling `BLOB` messages:
```
# hop publish kafka://scimma-server:9092/test-blob-file02 -f BLOB tests/data/example.gcn3
# hop subscribe kafka://scimma-server:9092/test-blob-file02 -s EARLIEST -p
Traceback (most recent call last):
  File "/opt/conda/bin/hop", line 10, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/hop/__main__.py", line 72, in main
    args.func(args)
  File "/opt/conda/lib/python3.7/site-packages/hop/subscribe.py", line 69, in _main
    print_message(message, args.json)
  File "/opt/conda/lib/python3.7/site-packages/hop/subscribe.py", line 29, in print_message
    print(str(message_model))
TypeError: __str__ returned non-string (type dict)
```

Reply:
Nice catch, I missed this in testing and it appears to be a blind spot in our CLI tests.